web/firefox_print_service.js

```diff
@@ -80,17 +86,29 @@ function FirefoxPrintService(pdfDocument, pagesOverview, printContainer) {
   this.pdfDocument = pdfDocument;
   this.pagesOverview = pagesOverview;
   this.printContainer = printContainer;
+  this._annotationStorage = pdfDocument.annotationStorage;
```

Comment:
In contrast to the print resolution from the other patch, the annotation storage is available on the `pdfDocument` variable that we already pass in everywhere, so why should we split this off? It's an extra variable we have pass everywhere and the number of function parameters is already quite high.

Reply:
I suppose that it's not *strictly* necessary, since the `AnnotationStorage`-instance is a bit of a special-case given that it's always unconditionally defined (whether that's a good idea is another question, but it's probably small enough to not really matter).

However, you obviously have a point about the unwieldy signatures (and that will get worse soon anyway...), so I'll just revert these changes; thank you!